roles/os_hardening/tasks/limits.yml

```diff
@@ -18,6 +18,7 @@
         limit_item: core
         value: "0"
         comment: Prevent core dumps for all users. These are usually not needed and may contain sensitive information
+      ignore_errors: "{{ ansible_check_mode }}"  # the directory needs to be created first, which does not happen in check_mode
```

Comment:
@rndmh3ro what do you think of this? 
alternatives are:
- create the directory as pre task
- add when condition to skip this in check mode